R/as_html.R

```diff
@@ -92,6 +96,8 @@ as_html <- function(x,
       cells[i, j][[1]] <- tagfun(
         class = if (inhdr) class_th else class_tr,
         class = if (j > 1 || i > nrh) paste0("text-", algn),
+        style = if (inhdr && !"header" %in% bold) "font-weight: normal;",
+        style = if (i == nrh && header_sep_line) "border-bottom: 1px solid black;",
```

Comment:
This is the top line right?

Reply:
Yes - the one inside the table (under the header)